erpnext/stock/report/stock_ledger/stock_ledger.py

```diff
@@ -194,8 +196,8 @@ def get_sle_conditions(filters):
 		conditions.append("voucher_no=%(voucher_no)s")
 	if filters.get("batch_no"):
 		conditions.append("batch_no=%(batch_no)s")
-	if filters.get("project"):
-		conditions.append("project=%(project)s")
+	if filters.get("package_tag"):
+		conditions.append("package_tag=%(package_tag)s")   # add condition to filter data by package tag
```

Comment:
- Why was project removed ?

Reply:
Hi @hrwX please check out this link:
https://app.asana.com/0/1200414497004131/1200402551564277/f

Reply:
it was mentioned to remove the filter for project, so removed it!

Reply:
@hrwX we are doing a cleanup of the reports - most of them look like data dumps and have columns that are not needed. Some additional reports are also constructed weirdly - in case of stock ageing removing a column would shift all the columns left (and they would be under wrong headers). I'm maintaining an active list of such reports - we'll fix them soon.